File: haitong/1_two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        mem = dict()
        for i in range(len(nums)):
            diff = target - nums[i]
            if diff in mem:
                return [mem[diff], i]
            mem[nums[i]] = i

File: xiaoting/1_twoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) { 
        int[] indices = new int[2];
        for(int i = 0; i < nums.length; i++){
            
            
            for(int m = i + 1; m < nums.length; m++){
                
                if(target - nums[i] == nums[m]){
                   
                    indices[0] = i;
                    indices[1] = m;
                }//end if
            }//end for
        }//end for
        
        return indices;
    }//end method
}